rules-topics/17_artificing.md

```diff
@@ -0,0 +1,198 @@
+   >5 Smithing Materials, 5 Alchemy Materials, 1 Spirit Gem.
+#### Wand-in-the-Box Trap
+> This Trap, designed by an expert in two disciplines, combines the efficacy of a Trap with the Magical ability of a Wand of Magic. Upon creating this Trap, the user must include a single Wand of Magic as crafting materials. When the Trap is activated, it delivers the effects of the wand, centered on the trap as "Voice Radius Magic [Spell Name]", at a casual speaking volume, such that someone 10 feet away could hear. This may not be used with Wands that have self-cast only spells, or the Destroy spell. This item is consumed on use. <br/>
+> 5 Smithing Materials, 5 Rune Carving Materials, 1 Spirit Gem. 1 Wand of Magic
+
+
+### Tier II
+
+#### Deep Breath Balm
+   >A special balm infused with magically treated herbs designed to clear the airways and enhance the user's breathing. Once per rest, the bearer may partake of these benefits to enhance their use of the Stamina ability Regeneration, reducing the concentration time from a minute to 30 seconds.<br/>
+   >15 Alchemy Materials, 5 Rune Carving Materials, 2 Spirit Gems.
+#### Destruction Magnet Enchantment
+   >This Enchantment may be placed upon a Shield, and while held will allow the user to re-direct any Destroy effect targeted at a different piece of equipment the bearer possesses to the Shield instead. The call for this is "Got it, Redirect, Resist". This Enchantment also gives the Shield 2x Ever Resist Destruction that stacks with any Hardening it may possess. After this enchantment has been applied, the shield may not be further Hardened. This enchantment may only be applied once.<br/>
+   >10 Alchemy Materials, 10 Smithing Materials, 2 Spirit Gems. 
+#### Magic Aura Enchantment
+   >This Enchantment may be placed upon a Weapon, and will change the weapon's base damage type to Magic.<br/>
+   >10 Rune Carving Materials, 10 Smithing Materials, 2 Spirit Gems.
+
+### Tier III
+
+####  Boisterous Bells
+   >These enchanted Bells are a Performer's best friend, especially when being magically heckled. These Bells will, two times per day, allow the user to spend a rank in Performance to Resist a Silence effect. The user does not regain the spent Performance ranks until after they complete a Rest.<br/>
+   >20 Rune Carving Materials, 10 Alchemy Materials, 3 Spirit Gems
+####  Mending Mallet
+   >A special Smithing Hammer built for a Smith on the go, with armor to repair and heads to smash. Three times per day, a user may spend their uses of Rapid Refit to refit the armor of another Individual, instead of themselves, by touching them by doing the usual 5 count and declaring "Rapid Refit". Only usable by an individual with Ranks in Smithing.<br/>
```

Comment:
Should we update this to say they can expend Stamina equal to the cost of Rapid Refit?